hbase-common/src/test/java/org/apache/hadoop/hbase/io/compress/CompressionTestBase.java

```diff
@@ -26,7 +26,7 @@
 import org.apache.hadoop.conf.Configurable;
 import org.apache.hadoop.hbase.util.Bytes;
 import org.apache.hadoop.hbase.util.EnvironmentEdgeManager;
-import org.apache.hadoop.hbase.util.RandomDistribution;
+import org.apache.hadoop.hbase.util.RandomDistributionCopy;
```

Comment:
Created a copy just to bring out the all the usages of the class: RandomDistribution to help decide where we move this in final patch.